dosenet.sh

```diff
@@ -12,7 +12,7 @@
 
 # setup paths and check config files
 HOME=/home/pi
-DOSENET=$HOME/dosenet-raspberrypi
+DOSENET=$HOME/dev/new_network/dosenet-raspberrypi
```

Comment:
Changing this path (and neither of us catching it before merging into master) is what caused all the devices to go down for a day a couple weeks ago. As we discussed, please clone `tybtab/dosenet-raspberrypi` into `$HOME/dosenet-raspberrypi` so that you can test without changing this path.